bundler/spec/install/process_lock_spec.rb

```diff
@@ -6,8 +6,8 @@
 
     it "will not run a second concurrent bundle install until the lock is released" do
       thread = Thread.new do
+        sleep 1 # ignore quality_spec
         Bundler::ProcessLock.lock(default_bundle_path) do
```

Comment:
I had to make this thread sleep first, otherwise it would error on line 15 that the lock was already held.